DetectorDescription/Detectors/compact/FCChh_HCal.xml

```diff
@@ -0,0 +1,39 @@
+<detector id="4" name="HCalBarrel0" type="CaloBarrel" readout="BarHCal_Readout" vis="hcal_envelope">
+  <dimensions rmin="BarHCal_rmin" rmax="BarHCal_rmax" dz="BarHCal_dz" />
+  <sequence_a id="0" name="sequence_1" material="Air" vis="hcal_barrel_seq1_vis">
+    <dimensions
+      dz="2*BarHCal_master_plate_thickness + BarHCal_spacer_plate_thickness + BarHCal_scintillator_thickness + 2*BarHCal_air_space_thickness"
+      dr="BarHCal_module_drho"
+      phiBins="BarHCal_n_phi_modules"
+      x="BarHCAL_module_spacing_phi"
+    />
+    <!-- Master plate -->
+    <module_component thickness="BarHCal_master_plate_thickness" y_offset="0" material="Steel235" sensitive="false" vis="hcal_steel_master_vis" />
+    <!-- Spacer plate -->
+    <module_component thickness="BarHCal_spacer_plate_thickness" y_offset="0" material="Steel235" sensitive="false" vis="hcal_steel_vis" />
+    <!-- Master plate -->
+    <module_component thickness="BarHCal_master_plate_thickness" y_offset="0" material="Steel235" sensitive="false" vis="hcal_steel_master_vis" />
+    <!-- not explicitly having air volumes here, instead adding offset to next volume -->
+    <!-- <module_component thickness="BarHCal_air_space_thickness" material="Air" sensitive="false" vis="hcal_air_vis" /> -->
+    <!-- Scintillator plate -->
+    <module_component thickness="BarHCal_scintillator_thickness" y_offset="BarHCal_air_space_thickness" material="Polystyrene" sensitive="true" vis="hcal_polystyrene_vis" />
+  </sequence_a>
+  <sequence_b id="1" name="sequence_2" material="Air" vis="hcal_barrel_seq2_vis">
+    <dimensions
+      dz="2*BarHCal_master_plate_thickness + BarHCal_spacer_plate_thickness + BarHCal_scintillator_thickness + 2*BarHCal_air_space_thickness"
+      dr="BarHCal_module_drho"
+      phiBins="BarHCal_n_phi_modules"
+      x="BarHCAL_module_spacing_phi"
+    />
+    <!-- Master plate -->
+    <module_component thickness="BarHCal_master_plate_thickness" y_offset="0" material="Steel235" sensitive="false" vis="hcal_steel_vis" />
+    <!-- Scintillator plate -->
+    <module_component thickness="BarHCal_scintillator_thickness" y_offset="BarHCal_air_space_thickness" material="Polystyrene" sensitive="true" vis="hcal_polystyrene_vis" />
+    <!-- Spacer plate -->
+    <module_component thickness="BarHCal_spacer_plate_thickness" y_offset="BarHCal_air_space_thickness"  material="Steel235" sensitive="false" vis="hcal_steel_vis" />
+    <!-- Master plate -->
+    <module_component thickness="BarHCal_master_plate_thickness" y_offset="0" material="Steel235" sensitive="false" vis="hcal_steel_vis" />
+  </sequence_b>
+  <end_plate name="end_plate" thickness="BarHCal_end_plate_thickness" material="Steel235" sensitive="false" vis="hcal_end_plate_vis" />
+  <face_plate name="face_plate" thickness="BarHCal_face_plate_thickness" material="Steel235" sensitive="false" vis="hcal_face_plate_vis" />
+</detector>
```

Comment:
This file seems to be a copy of HCal.xml. Maybe we should just do git mv instead of copying the file?